### PR TITLE
Tweaked Unbucklecuffing

### PR DESCRIPTION
**Pull request by Arrow768**
If you are buckled and cuffed to a chair it now takes 60 seconds longer to unbuckle yourself, but you are uncuffed at the same time.

If you use harm intent, you can break out of your cuffs in around a minute, but you will dislocate one of our arms in the process.

Implements: https://forums.aurorastation.org/topic/11366-revert-bucklecuffing-taking-two-resists/

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
60 seconds is pretty long. Two minutes is already forever if eyes are always on you, ready to drag you in a circle.

**Comment by Arrow768** [comment]
60 seconds longer is fine. You just need to time it correctly.

Alternatively you can always use the harm intent.

**Comment by Arrow768** [comment]
Your suggestion is out of the scope of this PR.

**Comment by dragoslava** [comment]
> Your suggestion is out of the scope of this PR.

Alrightie, aside from that I see no real issue with this

**Comment by skull132** [comment]
> 60 seconds is pretty long. Two minutes is already forever if eyes are always on you, ready to drag you in a circle.

If there are eyes on you, then you won't get out _anyways_. Making the countdown shorter will only result in more furious meme-matches between security and the assailant. 60 is, IMO, a decent midway point between, "FOREVER" and, "lmao let's go now".

File: code/modules/mob/living/carbon/resist.dm [hunk 1]
```diff
@@ -79,27 +79,58 @@
...
+		var/buckle_message_user = ""
+		var/buckle_message_other = ""
+		if(buckled) //If the person is buckled, also unbuckle the person
+			buckle_message_user = " and unbuckle yourself"
+			buckle_message_other = " and to unbuckle themself"
+			buckled.user_unbuckle_mob(src)
+
+		if(violent_removal)
+			var/obj/item/organ/external/E = H.get_organ(pick("l_arm","r_arm"))
+			E.dislocate(1)
```
Comment (skull132) [inline code/modules/mob/living/carbon/resist.dm]: Can `get_organ` return `null?` At which point, do a `if (!E) return` dealie before this.

**Pull request merged**